src/PostgREST/CLI.hs

```diff
@@ -205,6 +205,9 @@ exampleConfigFile =
       |# jwt-secret = "secret_with_at_least_32_characters"
       |jwt-secret-is-base64 = false
       |
+      |## Allows JWT caching to avoid validation on every request
+      |# jwt-caching = false
```

Comment:
Good thinking in making this false by default. For next major version we can make it true.